ngraph/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -385,7 +385,7 @@ void PostStepsList::add_convert_impl(const element::Type& type) {
         if (t == element::Type{}) {
             t = ctxt.target_element_type();
         }
-        if (t == node.get_node()->get_element_type()) {
+        if (t == node.get_node()->get_output_element_type(node.get_index())) {
```

Comment:
do we have more such places in prepro?

Reply:
I checked and didn't find any other names.
@nosovmik Please correct me if I am wrong

Reply:
yes, I used `node.get_element_type()` everywhere except this place. Will also implement unit test for this